Графика для всех страниц

DianaLad8 opened the pull request:
С иконками и кнопками пока не уверена. Размеры векторных рисунков, наверное, поменяются.
По семантике - дивы в подвале пока оставила, чтобы не сливалось в одну строку. В блоке "Результаты поиска" в списке найденных гостиниц под названием наоборот, пока оставила в одну строку.

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2189703/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2189703-sedona-36/3/

DianaLad8 committed:
Исправляет блок-герой, меняет секцию на статью-описание

После лайва-ретро некоторые уточнения в разметке

DianaLad8 committed:
Разбивает одну секцию на 2, убирает лишние дивы

В пагинации заменила ненумерованный список на нумерованный. Остальные дивы добавятся при оформлении стилей: блок с выпадающим списком

DianaLad8 committed:
Добавляет папку с рисунками в корневой каталог и вставляет контентные изображения на главной

DianaLad8 committed:
Убирает лишние спаны к иконкам

Подписи к иконкам переходят в атрибут alt

DianaLad8 committed:
Добавляет рисунки в каталог, убирает лишние подписи

Исправляет звездность гостиниц

DianaLad8 committed:
Добавляет комментарий

DianaLad8 committed:
Добавляет файл стилей и подключает ко всем страницам

Проверка с томатным цветом прошла)

DianaLad8 committed:
Подключает шрифты

В проекте Седона и в папке шрифтов только 2 шрифта - их и подключила

kamunicorn commented on catalog.html:
```diff
@@ -25,13 +26,16 @@
           <li class="navigaton-item">
             <a class="navigation-link" href="#">
-              <span class="visually-hidden">Поиск.</span>
+              <img src="images/search.svg" width="18" height="18" alt="Поиск.">
```
Это не контентное изображение и не изменяется в разных состояниях, поэтому в разметке его быть не должно.

kamunicorn commented on catalog.html:
```diff
@@ -45,16 +49,23 @@
           <li class="breadcrumbs-item">
-            <a class="breadcrumbs-link visually-hidden" href="index.html">Главная</a>
+            <a class="breadcrumbs-link visually-hidden" href="index.html">
+              <img src="images/home.svg" width="12" height="12" alt="Иконка домика - переход на главную страницу.">
```
Декоративный элемент. По стайлгайду он меняется, но это сделано через прозрачность, поэтому можно смело убирать его в css

kamunicorn commented on catalog.html:
```diff
@@ -45,16 +49,23 @@
+              <img src="images/home.svg" width="12" height="12" alt="Иконка домика - переход на главную страницу.">
+            </a>
+          </li>
+          <!--Это нужно будет убрать из разметки, как декоративный элемент!
```
про стрелку верно👍🏻

kamunicorn commented on catalog.html:
```diff
@@ -111,96 +122,85 @@ <h2 class="visually-hidden">Список гостиниц с фильтрами.
+          <li>
             <a class="button button-tile" href="#">
-              <span class="visually-hidden">Показать плитку.</span>
+              <img src="images/buttons-view/tile.svg" width="16" height="14" alt="Показать плитку.">
```
Тоже декоративное. В стайлгайде меняется только цвет бордера, а цвет самых иконок - нет.

kamunicorn commented on catalog.html:
```diff
@@ -111,96 +122,85 @@ <h2 class="visually-hidden">Список гостиниц с фильтрами.
           <li class="catalog-hotel">
             <a class="hotel-link" href="#">
-              <img class="hotel-img" src="#" alt="Очень красивая фотография.">
+              <img class="hotel-img" src="images/hotels/hotel-2.jpg" width="300" height="212" alt="Фотография горы на фоне серых туч с радугой.">
```
Хорошие описания👍🏻

kamunicorn commented on catalog.html:
```diff
@@ -111,96 +122,85 @@ <h2 class="visually-hidden">Список гостиниц с фильтрами.
-              <img src="#" alt="Звезда.">
-              <img src="#" alt="Звезда.">
-            </div>
+            <img src="images/star/star4.svg" width="140" height="37" alt="Четыре звезды.">
```
Не декоративный элемент. Там можно сделать все на css, хоть и немного сложно. Давай потом переделаешь.

kamunicorn replied:
Если заморачиваться меньше - можно вставлять и разные картинки звезд в css по количеству звезд. Но это будут 5 лишних файлов. Но можно обойтись всего одним файлом:)

kamunicorn commented on catalog.html:
```diff
@@ -237,17 +237,17 @@ <h2 class="visually-hidden">Социальные сети.</h2>
           <li class="footer-social-item">
             <a class="button-social" href="https://vk.com/htmlacademy">
-              <span class="visually-hidden">В контакте.</span>
+              <img src="images/button-social/vk.svg" width="24" height="14" alt="ВКонтакте.">
```
Это декоративное изображение, но сделать его только на css не получится: в стайлгайде у него есть состояния и он меняет цвет. Его надо вставлять куском svg в разметку.

kamunicorn commented on catalog.html:
```diff
@@ -259,7 +259,7 @@ <h2 class="visually-hidden">Социальные сети.</h2>
       <div class="footer-logo-academy">
         <a class="logo-academy" href=" https://htmlacademy.ru/">
-          <img  src="#"  alt="Логотип HTML Академии.">
+          <img  src="images/HTML Academy.svg" width="114.9" height="32.9"  alt="Логотип HTML Академии.">
```
Лучше переименовать без заглавных букв и пробелов, об этом и [критерий](https://up.htmlacademy.ru/htmlcss/36/criteries#PROJ-03) говорит. Нужно выработать такую привычку, чтобы потом не путаться, как там что написано, было единообразие и не возникало проблем с юниксом. Как раз можно проиллюстрировать проблему: поменяй в девтулз адрес картинки, заменю заглавную букву на строчную и картинка перестанет показываться. Такая штука на гитхабе, потому что он на юникс-системе.

kamunicorn commented on catalog.html:
```diff
@@ -237,17 +237,17 @@ <h2 class="visually-hidden">Социальные сети.</h2>
           <li class="footer-social-item">
             <a class="button-social" href="https://www.youtube.com/user/htmlacademyru">
-              <span class="visually-hidden">Ютуб.</span>
+              <img src="images/button-social/utube.svg" width="22" height="17" alt="Ютуб.">
```
youtube👀

kamunicorn commented on index.html:
```diff
@@ -42,11 +46,13 @@
-      </div>
+      <section class="hero" data-test="hero">
+        <h2 class="visually-hidden">Добро  пожаловать в Седону</h2>
+        <img src="images/index-background-hero.jpg" width="1200" height="485" alt="Фотография гор как фон.">
```
Фоновое изображение - в css для блока .hero

kamunicorn commented on index.html:
```diff
@@ -72,25 +78,28 @@ <h3 class="description-title">Мало туристов</h3>
         <p class="advantages-little-text">Опытный персонал и качественное обслуживание!</p>
         <ul class="advantages-list">
           <li class="advantages-item">
+            <img class="advantage-icon" src="images/advantage-icon/housing.svg" width="75" height="72" alt="Иконка жилья.">
```
декоративный элемент

kamunicorn reviewed (changes requested):
Декоративные элементы - убрать из разметки, оставить для css.
Соцсети в футере - вставить как svg в разметку.

DianaLad8 committed:
Убирает декоративные элементы, меняет иконки в подвале

Теги изображения у соцсетей в подвале меняет на тег векторного рисунка. Фоновый рисунок в блоке "hero" оставляет как контентный по желанию авторов курса

DianaLad8 committed:
Убирает фон из разметки

Блок "hero"
